rubocop-airbnb/lib/rubocop/cop/airbnb/continuation_slash.rb

```diff
@@ -14,7 +14,10 @@ def enforce_violation(node)
         alias on_send enforce_violation
         alias on_if enforce_violation
 
-        Util::ASGN_NODES.each do |type|
+        rubocop_version = Gem.loaded_specs.fetch('rubocop', nil)&.version.to_s
```

Comment:
Another way to load assignments would be: (but I think it's too aggressive)
```
begin
  assignments = RuboCop::Util::ASGN_NODES
rescue NameError
  assignments = RuboCop::AST::Node::ASSIGNMENTS
end
```

What do you think?

Reply:
I’m not actually sure what would be a better ruby idiom. perhaps @robotpistol or @BMorearty has thoughts?

Reply:
I should change the safe navigation operator to `try` but I'll wait for their response.
Thanks a lot @ljharb !

Reply:
doh yeah this is a rubocop version check, yeah same principles apply though. just throw in your strinafied version or just use the raw version and compare it to the `Gem::Version.new('2.6.0')` or whatever.

Reply:
Maybe I didn't get your solution but checking for the ruby version will fail if rubocop is updated but ruby isn't. `AST::Node::ASSIGNMENTS` was added in rubocop 0.59 which works for ruby > 2.2